src/test/javascript/e2e/entities/merchant.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+describe('Merchant e2e test', function () {
+
+    var username = element(by.id('username'));
+    var password = element(by.id('password'));
+    var entityMenu = element(by.id('entity-menu'));
+    var accountMenu = element(by.id('account-menu'));
+    var login = element(by.id('login'));
+    var logout = element(by.id('logout'));
+
+    beforeAll(function () {
+        browser.get('/');
+
+        accountMenu.click();
+        login.click();
+
+        username.sendKeys('admin');
+        password.sendKeys('admin');
+        element(by.css('button[type=submit]')).click();
```

Comment:
guess the above steps get performed in many tests. we should probably move them to a some sort of MockLogin object... not in the PR but...at somepoint